src/ontology/cob-edit.owl

```diff
@@ -1347,6 +1366,58 @@
         <obo:IAO_0000115>A process during which an organism comes into contact with another entity.</obo:IAO_0000115>
         <rdfs:label xml:lang="en">exposure of organism</rdfs:label>
     </owl:Class>
+
+
+
+    <!-- http://purl.obolibrary.org/obo/COB_0001100 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/COB_0001100">
+        <owl:equivalentClass>
+            <owl:Class>
+                <owl:unionOf rdf:parseType="Collection">
+                    <rdf:Description rdf:about="http://purl.obolibrary.org/obo/COB_0001101"/>
+                    <rdf:Description rdf:about="http://purl.obolibrary.org/obo/COB_0001102"/>
+                </owl:unionOf>
+            </owl:Class>
+        </owl:equivalentClass>
+        <rdfs:comment>This class groups the two defined classes &quot;disease&quot; and &quot;disorder&quot;. It supports use cases that conflate the two classes for pragmatic reasons.</rdfs:comment>
+        <rdfs:label xml:lang="en">disease or disorder</rdfs:label>
+        <rdfs:seeAlso rdf:resource="https://github.com/OBOFoundry/COB/pull/226"/>
+        <rdfs:seeAlso>https://github.com/OBOFoundry/COB/issues/19</rdfs:seeAlso>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/COB_0001101 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/COB_0001101">
+        <obo:IAO_0000115>A disposition (i) to undergo pathological processes that (ii) exists in an organism because of one or more disorders in that organism.</obo:IAO_0000115>
```

Comment:
@wdduncan suggests to make this a subclass of disposition in BFO. See related comment: https://github.com/OBOFoundry/COB/pull/226/files#r1090641271

Reply:
Hello Nico,
   Looks like we have a resolution. 
Three new terms in COB:
disease   COB_0001101
disorder  COB_0001102
disease or disorder  COB_0001100. - which is a union of COB_0001101 and COB_0001102

One request, for the source of the disease definition, that it be the Disease Ontology, as we originally defined the term 'disease' which OGMS then adopted. 
rdf:resource="http://purl.obolibrary.org/obo/doid.owl"/>

Cheers,
Lynn

Reply:
In OGMS, which is aligned with BFO, disease is defined as "A disposition (i) to undergo pathological processes that (ii) exists in an organism because of one or more disorders in that organism." This definition comes from [PMID:21347182](https://pubmed.ncbi.nlm.nih.gov/21347182/), "Toward an Ontological Treatment of Disease and Diagnosis," 2009, by Scheuermann, Ceusters, and Smith. This is original source of this definition (not DO).

Reply:
@matentzn see my comment [here](https://github.com/OBOFoundry/COB/pull/226#issuecomment-1409023666).